src/core/indexedlist.h

```diff
@@ -13,30 +13,91 @@ class IndexedList
     explicit IndexedList() = default;
```

Comment:
I still don't understand why `IndexedList` isn't equal to `QVector`, as far as I can tell `IndexedList` is basically a wrapper around `std::vector`, but `QVector` and `std::vector` are both a dynamically resized array.

I would be happy if somebody could explain this to me :)